### PR TITLE
Greenscraper

### PR DESCRIPTION
**Pull request by wrkode**
This PR addresses donation of GreenScraper to TAG Environmental Sustainability
It also adds a Table to the Repo for future projects.

### CODE REVIEW

File: greenscraper/LICENSE.md
Comment (leonardpahlke) [inline greenscraper/LICENSE.md:1]: we dont need this file. We will add it at project root.

**Review by leonardpahlke — COMMENTED** [review]
one comment otherwise LGTM

**Comment by leonardpahlke** [comment]
@wrkode across oss repos we usually require signing commits `git commit -s -m ""`, see: https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff.

**Comment by leonardpahlke** [comment]
ref: https://github.com/cncf-tags/tag-env-tooling/pull/1/checks?check_run_id=17829353133

**Comment by wrkode** [comment]
> ref: https://github.com/cncf-tags/tag-env-tooling/pull/1/checks?check_run_id=17829353133

Thanks @leonardpahlke, shall I cancel this PR and redo it?

**Comment by leonardpahlke** [comment]
You can modify the commit message. 


My sign off:
`Signed-off-by: leonardpahlke <leonard.pahlke@googlemail.com>`

You can run `git commit --amend --signoff -m "commit message"`

**Comment by leonardpahlke** [comment]
Thanks @wrkode !

**Pull request merged**